src/sass/components/ValidationProgress.scss

```diff
@@ -1,33 +1,92 @@
+@keyframes progressIndicator {
+  0% {
+    opacity: 0.4;
+    box-shadow: none;
+  }
+  50% {
+    opacity: 1;
+    box-shadow: 0 1px 2px -1px $color-primary;
+  }
+  100% {
+    opacity: 0.4;
+    box-shadow: none;
+  }
+}
+
 .ValidationProgress {
-  border-top: 1px solid $color-gray-lightest;
-  margin-top: 1em;
   padding-top: 1em;
   .usa-button {
     margin-top: .5em;
   }
-  .usa-unstyled-list>li {
-    margin-bottom: .5em;
+
+  .progressTotal {
+    background-color: $color-gray-lightest;
+    width: 100%;
+    height: 0.2em;
+    &::before {
+      content: "0";
+      font-size: 0.75em;
+      margin-top: -0.5em;
+      margin-left: 1.5em;
+      position: absolute;
+      left: 0;
+    }
+
+    &::after {
+      content: "100%";
+      font-size: 0.75em;
+      margin-top: -0.5em;
+      margin-right: -0.25em;
+      position: absolute;
+      right: 0;
+    }
   }
 
-  // circles as indicators
-  .progress {
-    background-color: $color-gray-light;
-    border-radius: 50%;
-    display: inline-block;
-    height: 10px;
-    margin-right: .25em;
-    width: 10px;
+  .progressFill {
+    background-color: $color-primary;
+    line-height: 1em;
+    height: 1em;
```

Comment:
I like the large progressFill, kinda lends more visual weight to what's been completed thus far.